omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -65,5 +61,4 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
 
     // bind functions to scope
     $scope.validateCondition = self.validateCondition;
-    $scope.unselectStatus = self.unselectStatus;
```

Comment:
Is this part of the ticket?

Reply:
yes, to remove the functionality of cross button 'x'